component-definition.json

```diff
@@ -602,6 +602,27 @@
               }
             }
           }
+        },
+        {
+          "title": "Carousel",
+          "id": "carousel",
+          "plugins": {
+            "xwalk": {
+              "page": {
+                "resourceType": "core/franklin/components/block/v1/block",
+                "template": {
+                  "name": "Carousel",
+                  "model": "carousel",
+                  "filter": "carousel",
+                  "item1": {
+                    "sling:resourceType": "core/franklin/components/block/v1/block/item",
+                    "name": "Cards",
+                    "model": "cards"
```

Comment:
I think we already have cards by default. Can we rename this into something else and maybe singular name?

Reply:
Fixed, added card items.